packages/ses/README.md

```diff
@@ -29,103 +29,121 @@ npm install ses
 
 ### Lockdown
 
-SES introduces the `lockdown` function.
-Alters the surrounding execution environment such that no object intrinsically
-accessible, like the array or function prototypes (`[].prototype` or `(() =>
-0).prototype`, can be replaced or subverted to attack other code in the same
-execution environment; and tames other intrinsically accessible utilities like
-regular expressions, random numbers, and clocks, to minimize opportunities for
-malicious programs to covertly infer the behavior of other programs running in
-the same execution environment.
+SES introduces the `lockdown()` function.
+Calling `lockdown()` alters the surrounding execution enviornment, or
+**realm**, such that no two programs running in the same realm can observe or
+interfere with each other until they have been introduced.
+
+To this end, `lockdown()` freezes all objects accessible to any program in the
+realm.
+The set of accessible objects includes but is not limited to: `globalThis`,
+`[].__proto__`, `{}.__proto__`, `(() => {}).__proto__` `(async () =>
+{}).__proto__`, the properties of any accessible object, the return values of
+any accessible function, and the result of using any accessible constructor.
```

Comment:
> the return values of any accessible function, and the result of using any accessible constructor.

Either I misunderstand you, or this is wrong. Calling a function or constructor can, and often does, allocate and return a fresh mutable object not locked down by `lockdown()`. I think just deleting that part of the sentence fixes the problem.